somatic_llr_filter.py

```diff
@@ -133,6 +146,17 @@ def create_vcf_writer(args, vcf_reader):
         od = OrderedDict([('ID', args.somatic_field), ('Number', '0'), ('Type', 'Flag'), ('Description', 'Is a somatic mutation')])
         new_header.add_info_line(od)
 
+    #check/add FILTER field in header
+    if not args.llr_threshold == None: #filtering may not even be specified
```

Comment:
I believe it's more idiomatic python to write:
```suggestion
    if args.llr_threshold is not None: #filtering may not even be specified
```